base/strings/string.jl

```diff
@@ -425,6 +425,17 @@ function reverse(s::String)
     String(buf)
 end
 
+"""
+    repeat(s::String, r::Integer)
+
+Repeat a string `r` times.
```

Comment:
well those two docstrings would probably be fine to list together since they're closely related - it's the array one that should be in a different section